diff --git a/node_modules/react-native-track-player/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt b/node_modules/react-native-track-player/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
index 13a6c60..09370d8 100644
--- a/node_modules/react-native-track-player/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
+++ b/node_modules/react-native-track-player/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
@@ -490,7 +490,7 @@ class MusicModule(reactContext: ReactApplicationContext) : NativeTrackPlayerSpec
 
         val indexInt = index.toInt()
         if (indexInt >= 0 && indexInt < musicService.tracks.size) {
-            callback.resolve(Arguments.fromBundle(musicService.tracks[indexInt].originalItem))
+            callback.resolve(Arguments.fromBundle(musicService.tracks[indexInt].originalItem ?: Bundle()))
         } else {
             callback.resolve(null)
         }
@@ -525,7 +525,7 @@ class MusicModule(reactContext: ReactApplicationContext) : NativeTrackPlayerSpec
         if (verifyServiceBoundOrReject(callback)) return@launchInScope
         callback.resolve(
             musicService.currentTrack?.let {
-                Arguments.fromBundle(it.originalItem)
+                Arguments.fromBundle(it.originalItem ?: Bundle())
             }
         )
     }
